openslides/core/static/templates/index.html

```diff
@@ -32,35 +32,91 @@
                                 </a>
                             </li>
                         </ul>
                     </div>
                     <div ng-if="!operator.isAuthenticated()">
-                        <a href="#" ng-click="showLoginForm = true" class="btn btn-default">
-                            <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-                            {{ gettext("Login") }}
+                        <a href="" class="btn btn-default" data-toggle="modal" data-target="#loginFormModal">
+                            <i class="fa fa-sign-in"></i>
+                            <translate>Login</translate>
                         </a>
-                        <div ng-if="showLoginForm">
-                            <form>
-                                username: <input type="text" ng-model="username"><br>
-                                password: <input type="password" ng-model="password"><br>
-                                <input type="submit" ng-click="login(username, password)" value="Save" />
-                            </form>
+                        <div class="modal" id="loginFormModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                            <div class="modal-dialog modal-sm">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                            <span aria-hidden="true">&times;</span>
+                                        </button>
+                                        <h4 class="modal-title" id="myModalLabel" translate>Login</h4>
+                                    </div>
+                                    <div class="modal-body">
+                                        <form>
+                                          <div class="form-group">
+                                            <label for="inputUsername" translate>Username</label>
+                                            <input type="text" ng-model="username" class="form-control input-lg" id="inputUsername">
+                                          </div>
+                                          <div class="form-group">
+                                            <label for="inputPassword" translate>Password</label>
+                                            <input type="password" ng-model="password" class="form-control input-lg" id="inputPassword">
+                                          </div>
+                                          <div class="form-group">
+                                              <button type="submit" ng-click="login(username, password)" class="btn btn-primary" translate>Login</button>
+                                              <!-- TODO: if anonymous user is activate -->
```

Comment:
You can access the config with the function ```config(CONFIG_NAME)``` so \<tag ng-if="config(FOO) == 'bar'"> should work